paddle/framework/operator.h

```diff
@@ -319,5 +319,14 @@ class OperatorWithKernel : public OperatorBase {
   virtual void InferShape(const InferShapeContext& ctx) const = 0;
 };
 
+namespace op_helpers {
+
+// Generate the gradient variable's name of a forward varialbe
+inline std::string GenGradName(const std::string& var) {
```

Comment:
work, but `GenGradName("")` is a little weird, but it can help to hide `GRAD_VAR_SUFFIX`.   I will follow this idea first.
@reyoung